src/modules/auth/services/users.service.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  BadRequestException,
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import * as bcrypt from 'bcrypt';
+import { Repository } from 'typeorm';
+import { User } from '../entities/user.entity';
+
+import type { AuthDto } from '../dtos/auth.dto';
+import type { CreateUserDto } from '../dtos/createUser.dto';
+import type { UpdateUserDto } from '../dtos/update.user.dto';
+import type { UserDto } from '../dtos/user.dto';
+
+@Injectable()
+export class UsersService {
+
+  constructor(
+    @InjectRepository(User) private readonly userRepo: Repository<User>,
```

Comment:
```suggestion
    private readonly _userRepo: UserRepository,
```

коли для кожнох сутності створиш окремий файл репозиторію